### ISSUE TITLE
Actions from reviewers tools are visible only after scrolling down

### ISSUE THREAD

**Issue by vcarciu** [mozilla/addons-frontend#4205]
Steps to reproduce:
1.Go to Reviewer Tools and open an add-on (https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/mixed-add-on)

Expected results:
A list of actions is displayed in the right side of the page

Actual results:
The actions are visible only after scrolling down the page

NOTES:
Issue was reproducible on Win10x64, FF 57

Please see video for this bug : 
![actionsrevtools](https://user-images.githubusercontent.com/1583842/35290431-ea4c243e-0072-11e8-92ed-2dee71478d9e.gif)

**Comment by tofumatt** [comment]
The reviewer tools are currently not part of the addons-frontend so this should be moved to addons-server.
